src/zepto.js

```diff
@@ -716,7 +713,7 @@ var Zepto = (function() {
         var element = this[0]
         if (typeof property == 'string') {
           if (!element) return
-          return element.style[camelize(property)] || getComputedStyle(element, '').getPropertyValue(property)
+          return element.style[camelize(property)] || getComputedStyle(element, '').getPropertyValue(dasherize(property))
```

Comment:
I just noticed that there is another place where we call `computedStyle.getPropertyValue(prop)` a few lines below, and that could benefit from the same fix that you did here. You would also need to add a new test for the `.css(["fooBar", "bazKuux"])` invocation style. Thanks!